hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestMetricsConnection.java

```diff
@@ -215,13 +234,13 @@ public void testStaticMetrics() throws IOException {
     metricKey = "rpcLocalExceptions_CallTimeoutException";
     counter = METRICS.getRpcCounters().get(metricKey);
     metricVal = (counter != null) ? counter.getCount() : 0;
-    assertTrue("metric: " + metricKey + " val: " + metricVal, metricVal == loop);
+    assertTrue("metric: " + metricKey + " val: " + metricVal, metricVal == loop * 2);
 
     // total exception
     metricKey = "rpcTotalExceptions";
     counter = METRICS.getRpcCounters().get(metricKey);
     metricVal = (counter != null) ? counter.getCount() : 0;
-    assertTrue("metric: " + metricKey + " val: " + metricVal, metricVal == loop * 2);
+    assertTrue("metric: " + metricKey + " val: " + metricVal, metricVal == loop * 3);
```

Comment:
Ditto. Please change to use assertEquals

Reply:
Resolved it.